graph/pkg/service/v0/graph.go

```diff
@@ -86,6 +88,14 @@ func (g Graph) GetHTTPClient() HTTPClient {
 	return g.httpClient
 }
 
+func (g Graph) publishEvent(ev interface{}) {
```

Comment:
Other services use the events interceptor to publish events. Imho better to do it the same way for all services. See here: https://github.com/cs3org/reva/blob/edge/internal/grpc/interceptors/eventsmiddleware/events.go

Reply:
That is not really possible since the requests aren't necessarily going through REVA.
I tried implementing it with a middleware in graph but that also doesn't really work since we need access to variables inside of the handler. This was the easiest way to implement the event stuff.